backend/substrapp/tasks/tasks.py

```diff
@@ -32,6 +32,9 @@
 PREFIX_HEAD_FILENAME = 'head_'
 PREFIX_TRUNK_FILENAME = 'trunk_'
 
+HASH_SALT_HEAD = 'HEAD'
+HASH_SALT_TRUNK = 'TRUNK'
```

Comment:
Following  
> This is to be consistent with the names of the arguments of the `get_hash` function. It doesn't have a concept of "salt", only of a key, so I fixed the inconsistency.

Is this still relevant to use `SALT` for those variables (and `_salt` references in the same file) ?

Reply:
I think so. Since it's something that is appended to the cache key, I think calling it "salt" makes sense. Do you have another idea in mind?

Reply:
`EXTENSION` or something in the same family :)